app/lib/dttp/code_sets/subject_specialisms.rb

```diff
@@ -0,0 +1,80 @@
+# frozen_string_literal: true
+
+module Dttp
+  module CodeSets
+    module SubjectSpecialisms
+      MAPPING = {
+        { name: "Graphic Design", marketing_name: "product design", allocation_subject: "Art and design" } => { entity_id: nil },
```

Comment:
Do we need the entity_id at all?
```suggestion
        { name: "Graphic Design", marketing_name: "product design", allocation_subject: "Art and design" },
```

Reply:
Probably not right now as i'll add it in my pr

Reply:
I'm also thinking to to restructure it so just the subject specialism name is the key only e.g.

```ruby
 "Creative Arts And Design" => { marketing_name: "creative arts and design", allocation_subject: "Art and design" }
```

This is consistent with all the other DTTP codesets.